python/src/spark_rapids_ml/core.py

```diff
@@ -586,6 +586,11 @@ def __init__(
         self.dtype = dtype
         self.n_cols = n_cols
 
+    def toHost(self) -> Model:
+        """Transform using pyspark ML model. For some reasons,
+        some algos can't be converted to pyspark ML Model"""
+        raise NotImplementedError()
```

Comment:
How about "Return the equivalent PySpark CPU model"?

And then, I'm wondering if we should rename `toHost()` to just `cpu()`, s.t. your example looks like:
```
model = LinearRegression().fit(xx)
# model.toHost().transform()
# model.toHost().predict()
model.cpu().transform()
model.cpu().predict()
```
... since `toHost()` makes me think of networking for some reason.  Not sure if anyone else has opinions on this?

Reply:
Really good suggestion, will change it accordingly.

Reply:
Host/Device terminology is everywhere in cuda.    

cupy and cudf use library names (asNumpy and to_pandas) to move things back and forth.  Though cupy also has the .get() to move off device.  

as_spark_ml?  A little long.    cpu is good.